### PR TITLE
Demo: Github integration with Jenkins

### PR DESCRIPTION
**Pull request by yuxinmmmm**
Demo: Github integration with Jenkins

### CODE REVIEW

File: contributions/demo/yuxin-urama/Readme.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Video Demo: Github integration with Jenkins
+
+## Members
+Ifeoma Urama (Urama@kth.se)
+
+Yuxin Meng (yuxinm@kth.se)
+
+## Outline
+1. Introduction of Jenkins as DevOps tool.
```
Comment (cesarsotovalero) [inline contributions/demo/yuxin-urama/Readme.md:9]: I recommend spending < 30s on this intro.

File: contributions/demo/yuxin-urama/Readme.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+2. Why use Jenkins with Github.
+3. Demo on how to integrate git with Jenkins
```
Comment (cesarsotovalero) [inline contributions/demo/yuxin-urama/Readme.md:11]: Make sure to explain the details and show an actual Github-Jenkins setup up and running.

**Comment by cesarsotovalero** [comment]
hi @yuxinmmmm, Looking forward to seeing your demo!

**Review by cesarsotovalero — COMMENTED** [review]
Consider my comments about your demo.

**Comment by bbaudry** [comment]
@yuxinmmmm please update the folder name with your kth user name (yuxinm)
thanks!

**Comment by monperrus** [comment]
ping @yuxinmmmm

**Comment by yuxinmmmm** [comment]
okay, I will update it soon :)

**Comment by yuxinmmmm** [comment]
I create a new pull request to update the folder name #1289

**Pull request merged**